SPTDataLoader/SPTDataLoaderRequestTaskHandler.m

```diff
@@ -162,9 +166,10 @@ - (NSURLSessionResponseDisposition)receiveResponse:(NSURLResponse *)response
 {
     self.response = [SPTDataLoaderResponse dataLoaderResponseWithRequest:self.request response:response];
     [self.requestResponseHandler receivedInitialResponse:self.response];
-    
+
+    NSHTTPURLResponse *httpResponse = nil;
```

Comment:
It's not clear why this local was moved.

Reply:
There was a check for response headers but that change was (only partially) reverted. Should be fixed now.